src/DataBaseReadOnly.vala

```diff
@@ -759,6 +794,34 @@ public class FeedReader.DataBaseReadOnly : GLib.Object {
 		return tags;
 	}
 
+	private Gee.List<string> read_taggings(string articleID)
+	{
+		var list = new Gee.LinkedList<string>();
+
+		var rows = m_db.execute("SELECT * FROM taggings WHERE articleID = ?", { articleID });
+
+		foreach(var row in rows)
+		{
+			list.add(row[1].to_string());
```

Comment:
Since we're only using one thing from the row, it would probably be better to explicitly list the column we want instead of doing a `SELECT *`